tests/test_async_with.py

```diff
@@ -131,7 +131,9 @@ async def test_create_pool_deprecations(mysql_params, loop):
             warnings.simplefilter("always")
             async with pool.get() as conn:
                 pass
-    assert issubclass(w[-1].category, DeprecationWarning)
+    # issubclass(w[-1].category, DeprecationWarning) fails on p3.7 with
+    # PYTHONASYNCIODEBUG=1, where 2 warnings are emitted.
```

Comment:
This refers to the code that is no longer there and so it'd be non-understandable by the reader. The comment must give an answer to the question "why?" without sending people to dig into history in order to find out what this mysterious `-1` (because it's not in the code next to the comment so what the comment is talking about).
```suggestion
    # The first warning emitted is expected to be DeprecationWarning:
    # in the past, we used to check for the last one but this assumption
    # breaks under Python 3.7 what also emits a `ResourceWarning` when
    # executed with `PYTHONASYNCIODEBUG=1`.
```